spec/yaml/swiftnav/sbp/tracking.yaml

```diff
@@ -18,7 +18,46 @@ include:
 definitions:
 
  - TrackingChannelState:
-    public: True
+    short_desc: Satellite tracking channel state
```

Comment:
Also, there was some notion of making `MsgObs`, `MsgEphemeris`, and `MsgBasePos` public, right? I think we're going to have to support stability on a message-by-message basis, which I think means fucking with the libsbp generator again.